mkdocs.yml

```
@@ -14,9 +14,10 @@ nav:
   - ehrQL: index.md
   - Tutorial:
     - Tutorial: tutorial/index.md
-    - Installation and setup of ehrQL: tutorial/installation-and-setup.md
+    - Installing ehrQL: tutorial/installing-ehrQL.md
+    - Generating dummy data: tutorial/generating-dummy-data.md
```

Just adding my ideas for heading here
```suggestion
    - Downloading dummy tables: tutorial/generating-dummy-data.md
```